package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
My bad, i should have been more clear, it is required by `grunt-remarkable` not by remarkable itself.
https://github.com/rwky/grunt-remarkable/blob/master/tasks/remarkable.coffee#L4
@vladikoff